docs/yul.rst

```diff
@@ -51,16 +51,16 @@ and ``mod`` are available either natively or as functions and computes exponenti
 .. code::
```

Comment:
Please also add in the paragraph above something like ``As per the warning above, the following code is untyped and can be compiled using `solc --strict-assembly` ``.